examples/beam/src/main/java/org/apache/nemo/examples/beam/AlternatingLeastSquare.java

```diff
@@ -264,6 +264,9 @@ public void processElement(final ProcessContext c) throws Exception {
    * Composite transform that wraps the transforms inside the loop.
    * The loop updates the user matrix and the item matrix in each iteration.
    */
+
+
+
```

Comment:
It seems like these few lines were added by mistake. Could you revert the changes for this file? Thanks! 😄